state/machine_linklayerdevices.go

```diff
@@ -923,59 +923,72 @@ func (m *Machine) LinkLayerDevicesForSpaces(spaces []string) (map[string][]*Link
 	if err != nil {
```

Comment:
I think the method's doc comment is incorrect now, since devices without addresses are being returned.